styles/cart.css

```diff
@@ -1,3 +1,105 @@
 body {
-  background-color: yellowgreen;
+  position: relative;
+  min-height: 100vh;
+}
+
+#main {
+  padding: 1% 4%;
+  margin: 2rem 0;
```

Comment:
You can consider using an utility class of margin vertical / y axis 
Since it'll be used at many places

Reply:
actually, there are variations at different places and hence can not use a generalized class everywhere